connect.go

```diff
@@ -69,7 +69,10 @@ func onRoutingNodeConnectionChanged(connected bool) {
 				return nil, nil
 			})
 			onAccountChanged()
+		} else {
+			updateNodeChannelPolicy(accData.Id)
```

Comment:
I am not sure it is the right point to start polling for updateNodeChannelPolicy. Here we are connected but not necessary have an opened channel.

Reply:
How about an else if accData.Status == data.Account_ACTIVE here?

Reply:
This still won't cover the case of fresh install, open channel, wait for confirmation and then trying to receive money.

Reply:
since the updateNodeChannelPolicy is using continues retries until it succeeds maybe we can call it anyway on start of on connect once with its own go routine.